webapp/app/assessment/components/dataTable/table/row.js

```diff
@@ -23,7 +23,7 @@ const Row = props => {
     return null
   }
 
-  const renderer = type === 'custom' ? render : rowRenderers[type]``
+  const renderer = type === 'custom' ? render : rowRenderers[type]
```

Comment:
maybe types.custom?

Reply:
removed completely. that was useless. we don't support custom type